napari/layers/points/_tests/test_points_mouse_bindings.py

```diff
@@ -12,19 +13,15 @@
 )
 
 
-@pytest.fixture
-def Event():
-    """Create a subclass for simulating vispy mouse events.
+@dataclass
```

Comment:
nice :-) we'll probably want this to be reusable in other files too one day, but that can come later